test/admin/deploy.test.js

```diff
@@ -18,8 +18,31 @@ describe('Deploy', () => {
 
   afterEach(async() => th.cleanup());
 
-  it('returns an empty list when requesting deploy history', async() => {
-    const deployHistory = await appDeploy.deployments().list();
-    expect(deployHistory).toEqual([]);
+  describe('when requesting the deploy history', () => {
+    it('returns a list containing the initial deploy', async() => {
+      const deployHistory = await appDeploy.deployments().list();
+      expect(deployHistory[0].status).toEqual('successful');
+    });
+  });
+
+  describe('when requesting the deploy config', () => {
+    it('returns the current config with an empty list of authenticated_repositories', async() => {
```

Comment:
nit: i think you should have a space between `async` and the parens. Maybe run prettier on this file?